File: reverse-arr.cpp
#include <iostream>
using namespace std;

int main() {
    int arr[] = {1, 2, 3, 4, 5};
    int n = sizeof(arr) / sizeof(arr[0]);

    // Print original array
    cout << "Original array: ";
    for (int i = 0; i < n; i++) {
        cout << arr[i] << " ";
    }

    // Reverse array
    for (int i = 0; i < n / 2; i++) {
        int temp = arr[i];
        arr[i] = arr[n - i - 1];
        arr[n - i - 1] = temp;
    }

    // Print reversed array
    cout << "\nReversed array: ";
    for (int i = 0; i < n; i++) {
        cout << arr[i] << " ";
    }

    return 0;
}

//my name is

File: reverse-arr.js
// reverse arr in javascript 

function reverseArray(arr) {
    let temp;
    let n= arr.length;
    for(let i= 0; i< arr.length/2; i++ ){
        temp = arr[i];
        arr[i] = arr[n-i-1]
        arr[n-i-1] = temp;
    }
    return arr;
}

console.log(reverseArray([1,2,3,4,5,6,7,8,9]));
